tasks/05_configure.yml

```
@@ -1,6 +1,14 @@
 ---
 - name: Add Borgmatic config file
   block:
+    - name: Register Borgmatic version
+      ansible.builtin.shell: "borgmatic --version | cut -d\\  -f2"
+      register: borgmatic_version
+      changed_when: False
+
+    - set_fact:
+        borgmatic_version="{{ borgmatic_version.stdout }}"
```

Hi, `ansible-lint` here complains that the task should be named, so I would suggest:

```suggestion
    - name: Update Borgmatic version fact
      ansible.builtin.set_fact:
        borgmatic_version: "{{ borgmatic_version.stdout }}"
```